cluster/engine_test.go

```diff
@@ -460,3 +472,23 @@ func TestDisconnect(t *testing.T) {
 	// Double disconnect shouldn't cause panic
 	engine.Disconnect()
 }
+
+func TestRemoveImage(t *testing.T) {
+	engine := NewEngine("test", 0, engOpts)
+
+	imageName := "test-image"
+	dIs := []types.ImageDelete{{Deleted: imageName}}
+
+	apiClient := engineapimock.NewMockClient()
+	apiClient.On("ImageRemove", context.TODO(), mock.Anything).Return(dIs, nil)
```

Comment:
I would rather the test args match what's being passed as close as possible. When the code changes to stop using the null context, we should reevaluate the test at that time as well.

I can see a different improvement to be made here.